components/org.wso2.carbon.identity.api.user.application/org.wso2.carbon.identity.rest.api.user.application.v1/src/main/java/org/wso2/carbon/identity/rest/api/user/application/v1/core/ApplicationService.java

```diff
@@ -83,6 +84,31 @@ public ApplicationResponse getApplication(String applicationId) {
         }
     }
 
+    /**
+     * Get application from application ID.
+     *
+     * @param applicationId unique identifier of the application
+     * @return a ServiceProvider instance.
+     */
+    public ServiceProvider getServiceProvider(String applicationId) {
```

Comment:
Move this method to https://github.com/wso2/identity-api-user/blob/master/components/org.wso2.carbon.identity.api.user.authorized.apps/org.wso2.carbon.identity.rest.api.user.authorized.apps.v2/src/main/java/org/wso2/carbon/identity/rest/api/user/authorized/apps/v2/core/AuthorizedAppsService.java

Reply:
Addressed in [81a8d70](https://github.com/wso2/identity-api-user/pull/129/commits/81a8d7074753b74f8bb43429db8d4ff80383675f)